.github/workflows/master_fuzzer_binaries.yml

```diff
@@ -35,8 +35,8 @@ jobs:
           sudo mkswap $SWAP_FILE
           sudo swapon $SWAP_FILE
 
-      - name: Install cargo fuzz subcommand crate
-        run: cargo install cargo-fuzz
+      - name: Install cargo bolero subcommand crate
+        run: cargo install cargo-bolero
 
       - run: rustup target add --toolchain nightly wasm32-unknown-unknown
```

Comment:
This can be removed, I believe – `rust-toolchain.toml` specifies this target as required and it’ll be installed automatically.

Reply:
Hmm does it when the `nightly` toolchain isn’t the one specified by rust-toolchain.toml? Or we could use a non-nightly rust with rustc_bootstrap, but that feels iffy :/

Reply:
Ah, I didn’t take note that this is nightly.